tensorflow/core/platform/default/build_config.bzl

```diff
@@ -152,7 +152,16 @@ def tf_additional_test_deps():
   return []
 
 def tf_additional_test_srcs():
-  return ["platform/default/test_benchmark.cc", "platform/posix/test.cc"]
+  return [
+      "platform/default/test_benchmark.cc",
+  ] + select({
+      "//tensorflow:windows" : [
+          "platform/windows/test.cc"
+        ],
+      "//conditions:default" : [
+          "platform/posix/test.cc",
```

Comment:
I think the same logic in CMake build is here:https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/cmake/tf_tests.cmake#L204